ansible/tasks/load_ddns_key_into_vault.yml

```diff
@@ -23,3 +23,5 @@
       data:
         ddns-key: "{{ bind9_ddns_key }}"
     body_format: json
+  vars:
```

Comment:
VENV containing vault module not needed to execute this ansible command.
We should remove this var from the execution of this task

Reply:
Done!